test/registry/registry_tests.py

```
@@ -3131,34 +3131,40 @@ def test_push_immutable_tag_blocked(
     """Test: Pushing to an immutable tag is blocked with a 409 error."""
     credentials = ("devtable", "password")
 
-    # Push an image to a new repository.
-    manifest_protocol.push(
-        liveserver_session, "devtable", "newrepo", "latest", basic_images, credentials=credentials
-    )
+    with FeatureFlagValue("IMMUTABLE_TAGS", True, registry_server_executor.on(liveserver)):
```

Ruff (0.15.0)

[error] `FeatureFlagValue` may be undefined, or defined from star imports

(F405)